python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -909,6 +909,9 @@ def test_resample():
     )
     expected = ser.resample("D").max()
     result = xser.resample("D").max()
+    # TODO: See if as_unit can be avoided
+    expected.index = expected.index.as_unit("s")
+    result.index = result.index.as_unit("s")
```

Comment:
Are these fixes related to `hash_array`, are they other fixes needed for cudf.pandas, or are they completely unrelated? If the second, could you update the PR title/description accordingly? If the third, can we remove from this PR?